### PR TITLE
copied layout from entry to comments

### LINKED ISSUE #36: Comment Alignment
### Background

In the [single post template](https://github.com/thoughtis/cata/blob/master/single.php) the [comments](https://github.com/thoughtis/cata/blob/master/comments.php) come after the [post content](https://github.com/thoughtis/cata/blob/master/template-parts/content/content.php)

The [entry component](https://github.com/thoughtis/cata/blob/master/assets/src/less/components/entry/layout.less) is synonymous with "post content" and it dictates the layout.

The comments aren't part of the post content and their layout is sort of undefined. See [the bottom of this post on Shop Catalog](https://shopcatalog.com/cute-face-masks-the-best-fashionable-face-masks-for-staying-safe/) for an example.

### Details

- [x] Update comments template to add `.comments__header` around `.comments__title` and `.comments__content` around the actual comments and the form.
- [x] Add a comments LESS component layout file that replicates the alignment of `.entry` and `.entry__content`
- [x] Create a beta release
- [x] Incorporate that beta into Shop Catalog locally so we can test it against the layout of Product Reviews which are technically comments and may share some class names.

### PR DESCRIPTION
**Pull request by richdacuban**
- Closes #36 
- Added `.comments` and `.comments__content` as wrappers around the title and content, respectively, in `comments.php`
- Added `assets/src/less/components/comments/layout.less` and duplicated the layout from `components/entry/layout.less` for the `.entry` and `.entry__content` classes

### CODE REVIEW

File: assets/src/less/components/comments/layout.less [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	> .alignfull {
+		max-inline-size: none;
+		.margin-inline( -@space-vw-2 );
+	}
+
+	> .woocommerce {
+		max-inline-size: 100%;
+	}
```
Comment (douglas-johnson) [inline assets/src/less/components/comments/layout.less:41]: I don't think we'll ever use these inside of the comments content. I should have been more clear in the issue. Sorry about that.

**Comment by douglas-johnson** [comment]
One extra request, it would be good if this could support center alignment too. By default Cata and its child themes are aligned to the left ( inline-start ). But I'm trying to keep the options open for child themes to be aligned the center. So, we can add a class to the child theme for `theme-prefers-align-center`. You can add it to the body in the browser to see what it does.

Can you update the comments layout to include...

```
.theme-prefers-align-center {
    .comments__title {
        .text-center;
    }
    .comments__content {
        > * {
            .margin-inline( auto );
        }
    }
}
```

File: assets/src/less/components/comments/layout.less (hunk 1 above)
Comment (richdacuban) [reply assets/src/less/components/comments/layout.less]: Np, cleared out.

**Comment by richdacuban** [comment]
> Can you update the comments layout to include...
> 
> ```
> .theme-prefers-align-center {
>     .comments__title {
>         .text-center;
>     }
>     .comments__content {
>         > * {
>             .margin-inline( auto );
>         }
>     }
> }
> ```

@douglas-johnson  Done

**Pull request merged**